subsys/nrf_rpc/rpc_utils/server/crash_gen.c

```diff
@@ -31,9 +34,11 @@ static void do_assert(void)
 
 static void do_hardfault(void)
 {
-	volatile uint32_t value;
-
-	value = 1 / 0;
+	__asm__ volatile (
```

Comment:
nit: the surrounding pragmas are no longer needed if you disable division by 0 in C.